<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DxWidget</class>
 <widget class="QWidget" name="DxWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPlainTextEdit" name="log">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="commandEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="sendButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="serverSelect">
       <property name="currentText">
        <string notr="true">db0sue.de:8000</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
       <item>
        <property name="text">
         <string>db0sue.de:8000</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="connectButton">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sendButton</sender>
   <signal>clicked()</signal>
   <receiver>DxWidget</receiver>
   <slot>send()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>504</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>connectButton</sender>
   <signal>clicked()</signal>
   <receiver>DxWidget</receiver>
   <slot>toggleConnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>696</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commandEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>DxWidget</receiver>
   <slot>send()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>send()</slot>
  <slot>toggleConnect()</slot>
 </slots>
</ui>
